assignments/pr_review_tutorial/chernyavskaya/README.md

```diff
@@ -1,13 +1,11 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+There are some two functions in simple_functions.py, "factorial" and "fibonacci".
+Factorial gives the factorial (the "!" function) of the argument:
 
     >>> from simple_functions import factorial
     >>> factorial(10)
-    9
+    3,628,800
 
-and this other part does something.  I forget why that I did it:
+Fibonacci gives you all the numbers in the Fibonacci sequence up to the argument:
```

Comment:
Descriptive description. Perhaps consider defining what a Fibonacci sequence is, but not incredibly necessary.